sdk/src/client/node_manager/syncing.rs

```diff
@@ -69,7 +66,7 @@ impl ClientInner {
 
         log::debug!("sync_nodes");
         let mut healthy_nodes = HashSet::new();
-        let mut network_nodes: HashMap<String, Vec<(ProtocolParameters, Node, Option<u64>)>> = HashMap::new();
+        let mut network_nodes: HashMap<String, Vec<(_, Node, Option<u64>)>> = HashMap::new();
```

Comment:
Just curious, why?

Reply:
Just didn't feel like importing it with the feature gate

Reply:
Fair enough